ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -50,7 +50,7 @@ class DemoListViewController: UITableViewController {
         // Fluent UI design recommends not showing "Back" title. However, VoiceOver still correctly says "Back" even if the title is hidden.
         navigationItem.backBarButtonItem = UIBarButtonItem(title: "", style: .plain, target: nil, action: nil)
 
-        tableView.backgroundColor = Colors.Table.background
+        tableView.backgroundColor = Colors.tableBackground
```

Comment:
Do we want to change [this](https://github.com/microsoft/fluentui-apple/blob/d16c8091a198293aacc893fb73e1d931e358b42b/ios/FluentUI/People%20Picker/PersonaListView.swift#L90) as well?